python/API_Examples_Python.py

```diff
@@ -0,0 +1,327 @@
+    order_response = requests.post(HOST_URL + "/v1/placeorder", post_data, headers=headers)
+    if order_response.status_code == requests.codes.ok:
+        pprint.pprint(order_response.json())
+    else:
+        print(order_response.content)
+        print("Failed to place order: ", order_response.json())
+
+
+def cancel_order():
+    """
+    cancel a single specific order
+    """
+    headers = {"Authorization": USER_INFORMATION["token"]}
+    pay_load = {"traderAddr": USER_INFORMATION["traderAddr"],
+                "orderId": "10827893",
+                "pairId": "ETH_ZIL",
+                "nonce": "2640819234038"}
+    pay_load = json.dumps(pay_load)
+    cancel_order_response = requests.post(HOST_URL + "/v1/cancelorder", pay_load, headers=headers)
+    if cancel_order_response.status_code == requests.codes.ok:
+        print("ok")
+    else:
+        print("Failed to cancel order: ", cancel_order_response.json())
+
+
+def cancel_all_orders():
+    """
+    Cancel all existing orders using given parameters
+    """
+    headers = {"Authorization": USER_INFORMATION["token"]}
+    pay_load = {"traderAddr": USER_INFORMATION["traderAddr"],
+                "pairId": 'ETH_ZIL',
+                "nonce": 10000000000000}
+    pay_load = json.dumps(pay_load)
+    resp = requests.post(HOST_URL + "/v1/cancelallorders", pay_load, headers=headers)
+    if resp.status_code == requests.codes.ok:
+        print(resp.content)
+    else:
+        print("Failed to cancel all orders: ", resp.json())
+
+
+def get_active_orders():
+    """
+    Get unfilled or partially filled orders that have not been cancelled or expired of a trader.
+    """
+    headers = {"Authorization": USER_INFORMATION["token"]}
+    url_params = "/v1/activeorders/" + USER_INFORMATION["traderAddr"] + "/ETH_ZIL/100/1"
+    resp = requests.get(HOST_URL + url_params, headers=headers)
+    if resp.status_code == requests.codes.ok:
+        pprint.pprint(resp.json())
+    else:
+        print("Failed to get active orders: ", resp.json())
+
+
+def get_trades():
+    """
+    Get recent trades on current wallet address.
+    """
+    headers = {"Authorization": USER_INFORMATION["token"]}
+    url_params = "/v1/trades/" + USER_INFORMATION["traderAddr"] + "/ETH_CHSB/100"
+    response = requests.get(HOST_URL + url_params, headers=headers)
+    if response.status_code == requests.codes.ok:
+        pprint.pprint(response.json())
+    else:
+        print("Failed to get trades: ", response.json())
+
+
+def get_signature(order_data):
+    """
+    The get_signature() function returns the signature for order signing
+    It takes in the order details for the input parameter
+    """
+
+    signature_hex = sign_order(order_data, USER_INFORMATION["privateKey"])
+    print('signature_hex:', signature_hex)
+    return signature_hex
+
+
+if __name__ == '__main__':
+
+    user_login()                      #This call returns a token, which should is placed in USER_INFORMATION["token"]
```

Comment:
Remove the comment if your implementation of user_login() does not return anything.

Reply:
Changed the user_login() function to return the token instead of modifying the global variable. Thanks for the explanation!